docs/source/quick-start.ipynb

```diff
@@ -11,7 +11,7 @@
   },
```

Comment:
<p>Shouldn't you be adding a <code>metric</code>  argument here? Random data was generated and then no metric applied. So could do <code>metric='mse'</code>  or something here plus <code>orientation=...</code>.  Maybe a comment in markdown above linking to the Delsole paper. </p><p> </p><p>Also the new docstring says for dim:</p><p> </p><pre class="ql-syntax" spellcheck="false">dimensions to apply metric to if metric is provided. Cannot contain

<code></code>time_dim<code></code>. Ignored if <code></code>metric<code></code> is None and observation is None.

Defaults to [].
</pre><p> </p><p>Although <code>dim='time'</code>  here references the time dim. Just looking for some clarity to new users here on what is being computed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/raybellwaves/xskillscore/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
True. Changed